consensus/polybft/consensus_runtime.go

```diff
@@ -309,21 +309,6 @@ func (c *consensusRuntime) populateFsmIfBridgeEnabled(
 		if err := c.state.cleanCommitments(nextStateSyncExecutionIdx); err != nil {
 			return fmt.Errorf("cannot clean commitments: %w", err)
 		}
-
-		nonExecutedCommitments, err := c.state.getNonExecutedCommitments(nextStateSyncExecutionIdx)
```

Comment:
Should we remove `func (s *State) getNonExecutedCommitments(startIndex uint64) ([]*CommitmentMessageSigned, error)` altogether? It is being used only in the unit test currently.

Reply:
Yes we can safely remove execution related things https://github.com/0xPolygon/polygon-edge/pull/964/commits/8a01208de461ddfdc7e5ac166096b505aba05e3b